packages/hardware-trezor/test/transformers/txIn.test.ts

```diff
@@ -20,4 +20,21 @@ describe('tx-inputs', () => {
       });
     });
   });
+  describe('mapTxIns', () => {
+    it('can map a a set of TxIns', async () => {
+      const txIns = await mapTxIns([txIn, txIn, txIn], CONTEXT_WITH_KNOWN_ADDRESSES);
+
+      expect(txIns.length).toEqual(3);
+
+      for (const input of txIns) {
+        expect(input).toEqual({
+          path: knownAddressKeyPath,
+          prev_hash: Buffer.from(txIn.txId).toString('hex'),
```

Comment:
Isn't this already correct format?
```suggestion
          prev_hash: txIn.txId,
```

Reply:
thanks @mkazlauskas 🙏  I still need to learn a lot about the correct formats in the SDK ;)